src/shared/components/table-grid/Table.scss

```diff
@@ -0,0 +1,13 @@
+.grid {
```

Comment:
This folder should be  named table not table-grid also name of class should be table not grid